examples/misc_exporter_gltf.html

```diff
@@ -417,6 +425,24 @@
 
 				scene1.add( object );
 
+				// ---------------------------------------------------------------------
+				// Empty buffer geometry
+				// ---------------------------------------------------------------------
+				var geometry = new THREE.BufferGeometry();
+				var numElements = 6;
+
+				var positions = new Float32Array( ( numElements ) * 3 );
+				var colors = new Float32Array( ( numElements ) * 3 );
+
+				geometry.addAttribute( 'position', new THREE.BufferAttribute( positions, 3 ) );
+				geometry.addAttribute( 'color', new THREE.BufferAttribute( colors, 3 ) );
+				geometry.setDrawRange( 0, 0 );
+
+				empty = new THREE.Mesh( geometry, new THREE.MeshBasicMaterial( { side: THREE.DoubleSide, vertexColors: THREE.VertexColors } ) );
+				empty.name = 'Custom buffered empty (drawrange)';
+				scene1.add( empty );
+
+
```

Comment:
What do you think about putting this case in `GLTFExporter.tests.js` instead? I think that might be a better fit than in the example.

Reply:
Yeah! I was thinking about it actually, but I didn't know we already had a `GLTFExporter.tests.js` 👼  I'll take a look and include it there thanks!